src/candidates/core/services/cv-parsing.service.js

```diff
@@ -0,0 +1,572 @@
+import similarity from 'string-similarity';
+
```

Comment:
Please skip to the definition of `fetchGoodMatchesIn`, for this is where the algorithm is implemented.

Note to self: these other utility functions could be extracted into the modules of their own.